curriculum/challenges/english/25-front-end-development/quiz-bash-scripting/66f1afbd9998e9c985d8e73b.md

```diff
@@ -17,439 +17,454 @@ Answer all of the questions below correctly to pass the quiz.
 
 #### --text--
 
-Placeholder question
+What is a correct shebang line in bash script?
 
 #### --distractors--
 
-Placeholder distractor 1
+`!#/bin/bash`
 
 ---
 
-Placeholder distractor 2
+`/#!bin/bash`
 
 ---
 
-Placeholder distractor 3
+`/#!/bin/bash`
 
 #### --answer--
 
-Placeholder answer
+`#!/bin/bash`
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which command would define `MY_VAR` and print it to the console?
 
 #### --distractors--
 
-Placeholder distractor 1
+`MY_VAR=10; echo MY_VAR`
 
 ---
 
-Placeholder distractor 2
+`$MY_VAR=10; echo MY_VAR`
 
 ---
 
-Placeholder distractor 3
+`$MY_VAR=10; echo $MY_VAR`
 
 #### --answer--
 
-Placeholder answer
+`MY_VAR=10; echo $MY_VAR`
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+How do you declare a numeric variable?
 
 #### --distractors--
 
-Placeholder distractor 1
+`declare -n MY_VAR=1`
 
 ---
 
-Placeholder distractor 2
+`declare -d MY_VAR=1`
 
 ---
 
-Placeholder distractor 3
+`declare -x MY_VAR=1`
 
 #### --answer--
 
-Placeholder answer
+`declare -i MY_VAR=1`
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which command would not run a `script.sh` file?
 
 #### --distractors--
 
-Placeholder distractor 1
+`./script.sh`
 
 ---
 
-Placeholder distractor 2
+`/tmp/script.sh`
 
 ---
 
-Placeholder distractor 3
+`bash script.sh`
 
 #### --answer--
 
-Placeholder answer
+`script.sh`
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What is the result of the command `type if`?
 
 #### --distractors--
 
-Placeholder distractor 1
+`if is a shell builtin`
 
 ---
 
-Placeholder distractor 2
+`if is a shell function`
 
 ---
 
-Placeholder distractor 3
+`if is a shell variable`
 
 #### --answer--
 
-Placeholder answer
+`if is a reserved word`
```

Comment:
I ran the command `type if` and got this answer:

![image](https://github.com/user-attachments/assets/6397639f-7c1d-4291-a17e-717e9dd9dbd5)

Reply:
![screenshot](https://github.com/user-attachments/assets/4628b816-370d-41fd-aaa0-118c769e3d83)

This is weird. Cause in my local Ubuntu terminal, it says the same thing this answer states.  If it helps, I'm using OMZ as the shell.

Reply:
It looks like output of this command is depends on used shell, i didn't notice it. I think it will be better to replace this question.

Reply:
I'd agree with you @dyadyaJora, it would be better to replace the question.